lib/betterdoc/containerservice/helpers/date_helper.rb

```diff
@@ -0,0 +1,35 @@
+module Betterdoc
+  module Containerservice
+    module Helpers
+      module DateHelper
+        extend ActiveSupport::Concern
+
+        def get_formatted_datetime(date)
+          return 'N/A' if date.nil? || date == 'N/A'
```

Comment:
The `N/A` value should be localized. `N/A` is fine for English but in German we would most likely use something different.

Reply:
Any suggestions? Since I don't speak German :)

Reply:
> Any suggestions? Since I don't speak German :)

Not really ;-) But I'm sure @BenLipinski as our UX specialist knows what we should display.

Reply:
So am I understanding correctly that we are looking for a german term for n/a (which is short for "not available" ?) ?

In that case "nicht verfügbar" would be a good German label. 

If possible, we should also provide the context. So if I get this correctly, it would be "Datum nicht verfügbar"